src/components/Card.js

```diff
@@ -15,15 +15,19 @@ export default class Card extends Component {
     const  { card, faceUp } = this.props
```

Comment:
It's a boolean that helps determine if the back of the card or front of the card should be displayed